wc-usage-tracking-auto-opt-in.php

```diff
@@ -35,3 +35,27 @@ function( $values ) {
 		return $values;
 	}
 );
+
+/**
+ * Bilmur RUM data collector
+ */
+add_action(
+	'wp_enqueue_scripts',
+	function() {
+		wp_enqueue_script( 'bilmur', 'https://s0.wp.com/wp-content/js/bilmur.min.js?m=202215', array(), '1.0.0', true );
```

Comment:
Is this a reliable spot to load this from? i.e., stored stably there on WPCOM?

I suspect it is but wanted to double check.

Reply:
Good call! The same script (different cache version only) was added to JetpackCRM website ([P2 here](https://avengersp2.wordpress.com/2021/10/22/request-adding-rum-to-jetpackcrm-com/)), and Jetpack CRM website ([P2](https://avengersp2.wordpress.com/2021/10/22/request-adding-rum-to-jetpackcrm-com/)), so I think we're safe.

But you're right that it's better to confirm and not assume, I'm gonna find a proper P2 to raise the question and post it here for reference.